src/client/src/ui/analytics/components/Analytics.tsx

```diff
@@ -103,7 +103,7 @@ function lastPositionWithDirection(lastPos: number) {
   }
   if (lastPos < 0) {
     color = 'red'
-    formattedLastPos = '-' + formattedLastPos
+    formattedLastPos = formattedLastPos
```

Comment:
Nice fix, I think you can just remove this line.

Reply:
Done !